src/components/Card.tsx

```diff
@@ -12,7 +12,7 @@ export type CardItem = {
 export default function Card({ title, link, description }: CardItem) {
   return (
     <div className="col col--4 margin-top--sm margin-bottom--md">
-      <Link className={clsx(styles.root, "card padding--sm")} href={link}>
+      <Link className={clsx(styles.root, "card")} href={link}>
```

Comment:
@subhamkumarr It looks like the homepage cards have small padding by default, so `padding--sm` is redundant and I can merge this change if you want. However, this doesn't quite fix the issue of cards being different heights – the card height still depends on the amount of text in the card + the width of your screen.

Reply:
By fixing this even i observe that the card height depends on the amount of text in the card. I look into all other card this change does not harm other card. 

I want this to be merge. This change making home page look beautiful.